php/libraries/Candidate.class.inc

```diff
@@ -681,6 +681,29 @@ class Candidate
         return $vLabel;
     }
 
+    /**
+     * Returns list of consents and their respective statuses for this candidate
+     *
+     * @return array List of consents and their associated values for this candidate.
+     *               The keys of the arrays are the IDs of the consents
+     */
```

Comment:
should put an endind dot at end of phrase to be consistant.

Reply:
i removed the full stop instead